src/search.rs

```diff
@@ -753,7 +802,8 @@ mod tests {
                 value: S("dolor sit amet, consectetur adipiscing elit"),
                 kind: S("text"),
                 number: 10,
-                nested: Nested { child: S("second") }
+                nested: Nested { child: S("second") },
+                _vectors: HashMap::from([(S("default"), vec![2000.0])]),
```

Comment:
Note to self/TODO:

`retrieveVectors` needs to be configurable and set to be able to get reuse the type
Docs: https://github.com/meilisearch/meilisearch/releases/tag/v1.9.0

Reply:
Do you want to do it here, or in another PR?

Reply:
Will do that here (would not make sense in another PR imo).
I am going to do that towards the end of the week